### PR TITLE
Atomic Roles Item Management - IR5, IR6, IR8

### PR DESCRIPTION
**Pull request by chinnu-focaloid**
Related to https://oat-sa.atlassian.net/browse/ADF-212

No | # | Role | Abilities | Includes | ACL
-- | -- | -- | -- | -- | --
9 | IR5 | Item resource creator | create item | IR4 | WRITE
10 | IR6 | Item importer | import single item & import package with several items | IR5 | WRITE
12 | IR8 | Item deleter | delete itemmove item | IR3 | WRITE

Also added Add File permission for content creator role

### CODE REVIEW

File: actions/class.ItemImport.php [hunk 1]
```diff
@@ -70,8 +71,21 @@ protected function getAvailableImportHandlers()
     private function replaceAvailableImportHandlers(): array
     {
         $returnValue = parent::getAvailableImportHandlers();
-
+        $hasAccess = $this->hasReadAccessByContext(
```
Comment (gabrielfs7) [inline actions/class.ItemImport.php:74]: Why no start this method with:

```
if (!$hasAccess) {
    return [];
}
```

Why iterate the loop if no importer should be allowed?

Comment (shpran) [reply actions/class.ItemImport.php:74]: I guess we no longer need this check and permission configuration (`isImporter`) - access rule for `taoItems_actions_ItemImport::index` will be enough.
WDYT @gabrielfs7 @chinnu-focaloid ?

Comment (chinnu-focaloid) [reply actions/class.ItemImport.php]: Done requested changes

**Review by shpran — APPROVED** [review]
Good job! 👍🏻 

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: migrations/Version202108110628142141_taoItems.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+namespace oat\taoItems\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\taoItems\model\user\TaoItemsRoles;
+use oat\tao\scripts\update\OntologyUpdater;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\tao\scripts\tools\accessControl\SetRolesAccess;
+
+final class Version202108110628142141_taoItems extends AbstractMigration
```
Comment (boajer) [inline migrations/Version202108110628142141_taoItems.php:31]: Any reason to have two independent migrations for one single feature\version?

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: migrations/Version202108030710512141_taoItems.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+use oat\taoItems\model\user\TaoItemsRoles;
+use oat\tao\scripts\update\OntologyUpdater;
+use oat\tao\model\accessControl\ActionAccessControl;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\tao\scripts\tools\accessControl\SetRolesAccess;
+
+final class Version202108030710512141_taoItems extends AbstractMigration
+{
+    private const CONFIG = [
+        SetRolesAccess::CONFIG_RULES => [
```
Comment (gabrielfs7) [inline migrations/Version202108030710512141_taoItems.php:35]: It is missing some permissions here: 

https://github.com/oat-sa/extension-tao-item/pull/500/files#diff-29bd6b821eb60fe0c4f53047f9a4ca01847292eb47203f88f7ee910bf81841b6R197

File: migrations/Version202108110628142141_taoItems.php (hunk 2 above)
Comment (gabrielfs7) [reply migrations/Version202108110628142141_taoItems.php:31]: Yes, just saw it now. https://github.com/oat-sa/extension-tao-item/pull/500/files#r686564952

Lets create a single migration instead

File: migrations/Version202108110628142141_taoItems.php [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+{
+    private const CONFIG = [
+        SetRolesAccess::CONFIG_RULES => [
```
Comment (gabrielfs7) [inline migrations/Version202108110628142141_taoItems.php:34]: I did the same in my branch, cause it was blocking my task 

https://github.com/oat-sa/extension-tao-item/pull/502/files#diff-e4657282dcf7c90a48b9211d94cd9d46d9de62ce3defe4b76645248999dffef5R16

Please let me know when it get merged

Comment (chinnu-focaloid) [reply migrations/Version202108110628142141_taoItems.php]: Done changes

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Comment by Natallia-Radkevich** [comment]
QA verified on  http://test-chinnufrancis.playground.kitchen.it.taocloud.org:48831/tao/Main/login

**Pull request merged**